index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Tribute Page</title>
+    <link rel="stylesheet" type="text/css" href="./css/styles.css" />
+  </head>
+  <body>
+    <div id="main">
+      <div id="title"><h1>Dr. Norman Borlaug</h1></div>
```

Comment:
It would be nice to make the `div` here also [a `header` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header), this would make it more semantic compare to a `div` tag.